apps/src/calc/calc.js

```diff
@@ -716,6 +724,7 @@ function isPreAnimationFailure(testResult) {
     testResult === TestResults.EMPTY_FUNCTIONAL_BLOCK ||
     testResult === TestResults.EXTRA_TOP_BLOCKS_FAIL ||
     testResult === TestResults.EXAMPLE_FAILED ||
+    testResult === TestResults.APP_SPECIFIC_FAIL || // TODO(bcjordan): is this wrong?
```

Comment:
Though it might hold true right now, one could conceive that there would later be app specific failures that are not preanimation failures. I guess if we wanted to be more vigilant we could also check message? It's perhaps a little weird, because some of these other errors could also have been type of APP_SPECIFIC_FAIL instead